src/ranged.cpp

```diff
@@ -948,43 +943,42 @@ static void do_aim( player *p, std::vector <Creature *> &t, int &target,
         p->recoil = std::max(MIN_RECOIL, p->recoil);
     }
 
-    const int aim_amount = p->aim_per_time( *relevant, p->recoil );
+    const double aim_amount = p->aim_per_move( *relevant, p->recoil );
     if( aim_amount > 0 ) {
         // Increase aim at the cost of moves
-        p->moves -= 10;
+        p->moves--;
```

Comment:
I'm considering a reworking of the code so that it's not dependent upon `sight_dispersion`. The aiming options I want to provide are:

* **snapshot** - no aiming at all
* **aimed** - 50% chance of `accuracy_goodhit`
* **maximum** - maximum aiming

Any others?